src/screens/TopicMemberScreen/index.js

```diff
@@ -228,13 +180,17 @@ const TopicMemberScreen = () => {
         onCancelToken={onTokenCancel}
         placeholderText={StringConstant.topicMemberPlaceholder}
         setIsFirstTimeOpen={setIsFirstTimeOpen}
-        hideBackIcon={true}
+        getSearchLayout={setSearchHeight}
       />
       <ScrollView
+        decelerationRate="fast"
+        scrollEventThrottle={1}
         style={styles.fragmentContainer}
-        contentContainerStyle={styles.fragmentContentContainer}
+        contentContainerStyle={[styles.fragmentContentContainer, {paddingTop: headerShowHeight}]}
         keyboardShouldPersistTaps="handled"
-        onScroll={handleScrollEvent}>
+        keyboardDismissMode="on-drag"
+        removeClippedSubviews={true}
+        onScroll={handleScroll}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Several new props have been added to the `ScrollView` component (`decelerationRate`, `scrollEventThrottle`, `keyboardDismissMode`, `removeClippedSubviews`, `onScroll`). Make sure these props are correctly implemented and do not negatively impact the behavior of the `ScrollView`.
```diff
+        decelerationRate="fast"
+        scrollEventThrottle={1}
+        keyboardDismissMode="on-drag"
+        removeClippedSubviews={true}
+        onScroll={handleScroll}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->